### PR TITLE
CEXT-2829: Draft of webhooks admin UI docs page

### PR DESCRIPTION
**Pull request by ryanjcohen**
## Purpose of this pull request

This pull request (PR) adds a draft documentation page for webhooks admin UI functionality.

## Affected pages

- ...

## Links to Magento Open Source code

- ...

### CODE REVIEW

File: src/pages/webhooks/admin-configuration.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: Admin configuration
+description: Learn how to extend, override, and create hooks using the Webhooks Admin UI pages.
+keywords:
+  - Extensibility
+---
+
+# Admin configuration
+
+The Commerce Admin UI can be used to extend and override existing hooks defined in `webhooks.xml` files and to create new hooks. Using the Admin UI, webhook configuration changes can be made without XML file updates, and, for Cloud instances, redeployment. Configuration changes made using the Commerce Admin will impact webhook execution and the output of the `webhooks:list` command.
```
[the remark this answers is not in the document]
Comment (keharper) [reply src/pages/webhooks/admin-configuration.md:10]: How so?

File: src/pages/webhooks/admin-configuration.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## View registered hooks
+
+A grid listing the registered hooks can be found in the Commerce Admin:
```
Comment (keharper) [inline src/pages/webhooks/admin-configuration.md:14]: Delete this line

File: src/pages/webhooks/admin-configuration.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+*  In the Admin, select **System** > Webhooks > **Webhooks** to display the _Webhooks_ grid page.
+
+![Webhooks grid](../_images/webhooks/webhooks-grid.png)
+
+The rows of this grid show configuration settings for all registered hooks, both active and inactive.
+
+## Edit an existing hook
+
+A form for editing existing hooks listed in the webhooks grid can be accessed by clicking **Edit** in the **Action** column of the grid.
```
Comment (keharper) [inline src/pages/webhooks/admin-configuration.md:24]:
```suggestion
Click **Edit** in the **Action** column to modify an existing hook.
```

File: src/pages/webhooks/admin-configuration.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+![Edit hook settings](../_images/webhooks/edit-hook-settings.png)
+
+In the **Hook settings** section of the form, hook attributes other than the hook name and names of the associated webhook method, webhook type, and batch can be modified. For a hook defined in a `webhooks.xml` file, the **Active** toggle can be used to override the value of the hook's `remove` attribute.
```
Comment (keharper) [inline src/pages/webhooks/admin-configuration.md:28]:
```suggestion
You can fully edit any hook that was originally defined in the Admin. However, hooks defined in a `webhooks.xml` file are only partially editable. For these hooks, you cannot change the value of the webhook method, webhook type, batch name, or hook name. You can use the **Active** toggle to override the value of the `remove` attribute for any hook defined in an XML file.
```

File: src/pages/webhooks/admin-configuration.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+The **Hook fields**, **Hook Headers**, and **Hook Rules** form subsections can be used to edit or add hook fields, headers, and rules, respectively. For fields, headers, and rules defined in `webhooks.xml` files, the values of certain attributes cannot be changed through the form.
+
+![Edit hook fields](../_images/webhooks/edit-hook-fields.png)
+
+## Create a new hook
+
+A form for creating a new hook can be accessed by clicking **Add New Webhook** from the grid page. If the plugin for the webhook method entered into the form has not been generated for the Commerce instance, a warning to run the `webhooks:generate:module` command will appear upon clicking **Save**.
```
Comment (keharper) [inline src/pages/webhooks/admin-configuration.md:36]:
```suggestion
Click **Add New Webhook** from the grid page to display the form for creating a new hook. If the plugin for the webhook method entered into the form has not been generated for the Commerce instance, a warning to run the `webhooks:generate:module` command will appear upon clicking **Save**.
```

Comment (keharper) [reply src/pages/webhooks/admin-configuration.md:36]: Move the create section to be above the editing section.

**Review by keharper — CHANGES_REQUESTED** [review]

File: src/pages/webhooks/admin-configuration.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ryanjcohen) [reply src/pages/webhooks/admin-configuration.md]: I think there's a typo in the added suggestion - the "XML"

**Review by keharper — APPROVED** [review]

**Pull request merged**